src/System.Reflection.Context/src/Configurations.props

```diff
@@ -1,11 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
       netstandard;
       netstandard1.1;
-      uap;
-      _netfx;
+      netstandard2.1;
```

Comment:
Also, you probably are missing a netstandard 2.1 configuration for this in the ref, or at least it looks that way by seeing the error you hit at CI.